src/activity-logger/background/log-page-visit.js

```diff
@@ -1,8 +1,10 @@
 import db from 'src/pouchdb'
 import storePage from 'src/page-storage/store-page'
+import { generatePageDocId } from 'src/page-storage'
 import { checkWithBlacklist } from 'src/blacklist'
 import { generateVisitDocId } from '..'
 import * as index from 'src/search'
+import { initSingleLookup } from 'src/search/search-index/util'
```

Comment:
This should be available already on the `index` object from the previous import (`import * as index from 'src/search'`)